### PR TITLE
Update playbooks example to install nmstate on target machine

### LINKED ISSUE #15: Update the example in playbooks to install nmstate on the target machine
Currently, `./playbooks/eth1_ip.yml` required nmstate installed on the virtual machine first. Instead of that, we want to update this playbooks to install nmstate first before running any other tasks.

### PR DESCRIPTION
**Pull request by liangwen12year**
Signed-off-by: Wen Liang <liangwen12year@gmail.com>

### CODE REVIEW

File: playbooks/eth1_ip.yml [hunk 1]
```diff
@@ -2,6 +2,18 @@
 - hosts: all
   become: True
   tasks:
+    - name: Install nmstate when ansible distribution is 'CentOS'
+      command:
+        cmd: yum install nmstate -y
+        warn: false
```
Comment (tyll) [inline playbooks/eth1_ip.yml]: Why is this needed?

Comment (liangwen12year) [reply playbooks/eth1_ip.yml]: install nmstate on target machine

Comment (tyll) [reply playbooks/eth1_ip.yml]: Ok, but why the `warn: false`?

Comment (liangwen12year) [reply playbooks/eth1_ip.yml]:
```TASK [Install nmstate when ansible distribution is 'CentOS'] **************************************************************************************************************************************************
task path: /home/wenliang/ansible-nmstate/playbooks/eth1_ip.yml:5
[WARNING]: Consider using the yum module rather than running 'yum'.  If you need to use command because yum is insufficient you can add 'warn: false' to this command task or set 'command_warnings=False' in
ansible.cfg to get rid of this message.```

Comment (liangwen12year) [reply playbooks/eth1_ip.yml]: I use `warn: false`  to get rid of warning message

Comment (tyll) [reply playbooks/eth1_ip.yml]: Why is this the correct way to handle the warning?

Comment (liangwen12year) [reply playbooks/eth1_ip.yml]: This is probably permissive, the above `cmd: yum install nmstate -y`  that would generate a warning would no longer generate a warning.

Comment (tyll) [reply playbooks/eth1_ip.yml]: What about the other option that the warning suggests? Why not use that one?

Comment (liangwen12year) [reply playbooks/eth1_ip.yml]: I have not try the yum module, but it seems that using yum module is better. I will update soon !

Comment (liangwen12year) [reply playbooks/eth1_ip.yml]: @tyll ,updated. Thanks for the hint!

Comment (tyll) [reply playbooks/eth1_ip.yml]: FYI: There is also a generic package module that would work for even more distribtuions: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/package_module.html

Comment (liangwen12year) [reply playbooks/eth1_ip.yml]: We will change from `dnf` module to `package` module, because it works on more distributions

**Review by ffmancera — APPROVED** [review]
Looks good to me. Thanks!

**Review by tyll — APPROVED** [review]
Thank you, :+1:
